build_config/serenity-32bit.rb

```diff
@@ -0,0 +1,12 @@
+MRuby::CrossBuild.new('serenity-32bit') do |conf|
```

Comment:
SerenityOS is a relatively less known operating system yet. So it seems better to have a short description of the OS at the top of the configuration file.

Besides that, the configuration name is `serenity-32bit`. Do you (or they) have any plan to support non-32bit configuration on SerenityOS (maybe in the future)? If not, it should be just `serenity` (or `cross-serenity` if you plan to build on the serenity host).

Reply:
I've added a short description to the top of the file.

I also replaced the 32bit specific version with one that can compile on 64bit systems. 64bit is new in Serenity so I had to check with the developers to see how to get the correct platform.

Thank you! :)

Reply:
Thank you. Could you tell me if `SERENITY_ARCH` is a common environment variable with Serenity development?
If so, it's OK. Otherwise, could you add a description of the variable?

Reply:
`SERENITY_ARCH` is defined by the SerenityOS [ports system](https://github.com/SerenityOS/serenity/blob/master/Ports/.port_include.sh#L5), so it should always be there. I can add an extra comment specifying the default value just in case.